cmd/expired-authz-purger/main.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2014 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/cactus/go-statsd-client/statsd"
+	"github.com/codegangsta/cli"
+	"github.com/jmhodges/clock"
+	"gopkg.in/gorp.v1"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type expiredAuthzPurger struct {
+	stats statsd.Statter
+	log   blog.Logger
+	clk   clock.Clock
+	db    *gorp.DbMap
+
+	batchSize int64
+}
+
+func (p *expiredAuthzPurger) setDefaults() {
+	if p.batchSize == 0 {
```

Comment:
You could use a `cli.IntFlag` with a default value for this and set it in the constructor below instead of having a separate method.